cmake/arm64x.cmake

```diff
@@ -0,0 +1,33 @@
+set(arm64ReproDir "${CMAKE_SOURCE_DIR}/repros")
+
+if("${BUILD_AS_ARM64X}" STREQUAL "ARM64")
+	foreach (n ${ARM64X_TARGETS})
+		add_custom_target(mkdirs_${n} ALL COMMAND cmd /c (if exist \"${arm64ReproDir}/${n}_temp/\" rmdir /s /q \"${arm64ReproDir}/${n}_temp\") && mkdir \"${arm64ReproDir}/${n}_temp\" )
```

Comment:
These commands look hacky. I'd prefer to discuss it offline with you to understand why these changes are needed.